readme.md

```diff
@@ -59,7 +59,7 @@ GLuint CreateTexture(char const* Filename)
 
 | Service | System | Compiler | Status |
 | ------- | ------ | -------- | ------ |
-| [Travis CI](https://travis-ci.org/g-truc/gli)| Linux 64 bits | Clang 3.4, Clang 3.8, GCC 4.9 | [![Travis CI](https://travis-ci.org/g-truc/gli.svg?branch=master)](https://travis-ci.org/g-truc/gli)
+| [Travis CI](https://travis-ci.org/g-truc/gli)| Linux 64 bits | Clang 7.0, Clang 9.0, GCC 7.4, GCC 9, GCC 10 | [![Travis CI](https://travis-ci.org/g-truc/gli.svg?branch=master)](https://travis-ci.org/g-truc/gli)
```

Comment:
The mentioned extra compiler compatibility. The `.travis.yml` config added to the PR provides configurations for these compilers.